cmd/flush_handles/flush_handles_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"flag"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-tpm-tools/simulator"
+	"github.com/google/go-tpm-tools/tpm2tools"
+	"github.com/google/go-tpm-tools/tpm2tools/tpm2toolstest"
+	"github.com/google/go-tpm/tpm2"
+)
+
+const (
+	// Maximum number of handles to keys tests can create within a simulator.
+	maxHandles = 3
+)
+
+func TestHandleTypesFromFlags(t *testing.T) {
+	for _, test := range []struct {
+		flushTransientFlag     string
+		flushLoadedSessionFlag string
+		flushSavedSessionFlag  string
+		flushAllTypesFlag      string
+		want                   []tpm2.HandleType
+	}{
+		{"f", "f", "f", "f", []tpm2.HandleType{}},
+		{"t", "f", "f", "f", []tpm2.HandleType{tpm2.HandleTypeTransient}},
+		{"f", "t", "f", "f", []tpm2.HandleType{tpm2.HandleTypeLoadedSession}},
+		{"f", "f", "t", "f", []tpm2.HandleType{tpm2.HandleTypeSavedSession}},
+		{"f", "t", "t", "f", []tpm2.HandleType{tpm2.HandleTypeLoadedSession, tpm2.HandleTypeSavedSession}},
+		{"t", "t", "f", "f", []tpm2.HandleType{tpm2.HandleTypeTransient, tpm2.HandleTypeLoadedSession}},
+		{"t", "f", "t", "f", []tpm2.HandleType{tpm2.HandleTypeTransient, tpm2.HandleTypeSavedSession}},
+		{"t", "t", "t", "f", []tpm2.HandleType{tpm2.HandleTypeTransient, tpm2.HandleTypeLoadedSession, tpm2.HandleTypeSavedSession}},
+		{"f", "f", "f", "t", []tpm2.HandleType{tpm2.HandleTypeTransient, tpm2.HandleTypeLoadedSession, tpm2.HandleTypeSavedSession}},
+	} {
+		// Overwrite flag pointer values.
+		flag.Set("flush-transient", test.flushTransientFlag)
+		flag.Set("flush-loaded-session", test.flushLoadedSessionFlag)
+		flag.Set("flush-saved-session", test.flushSavedSessionFlag)
+		flag.Set("flush-all-types", test.flushAllTypesFlag)
+
+		got := handleTypesFromFlags()
+		if !reflect.DeepEqual(got, test.want) {
+			t.Fatalf("got %v; want %v;", got, test.want)
+		}
+	}
+}
+
+func TestFlush(t *testing.T) {
+	simulator, err := simulator.Get()
```

Comment:
Use `internal.GetTPM()` see [this example](https://github.com/google/go-tpm-tools/blob/551cd9014fbdf21ed8bbe3e1a5cb7ec1564c3c2d/tpm2tools/keys_test.go#L30-L32)

Reply:
Done.